src/shared/sam/cli/samCliInvokerUtils.ts

```diff
@@ -44,7 +39,7 @@ export class DefaultSamCliUtils {
 }
 
 export class InvalidSamError extends Error {
-    public constructor (public versionValidation: SamCliVersionValidatorResult) {
+    public constructor(public versionValidation: SamCliVersionValidatorResult) {
```

Comment:
is there linting for these sorts of whitespace changes to ensure consistency ?

Reply:
We have been relying on the VS Code formatter. @mpiroc is this something the linter could assist with?

Reply:
We do--see `tslint.yaml` in the repo root:

```yaml
  space-before-function-paren:
    options:
      -
        anonymous: never
        named: never
        method: never
        contructor: never
```

We also lint in the CI builds, so I'm not sure how this space ended up here.